spoon-runner/src/main/java/com/squareup/spoon/SpoonUtils.java

```diff
@@ -87,11 +87,25 @@ static void setDdmlibInternalLoggingLevel() {
     DdmPreferences.setLogLevel("debug");
   }
 
-  /** Find all device serials that are plugged in through ADB. */
-  public static Set<String> findAllDevices(AndroidDebugBridge adb) {
+  /**
+   * Find all device serials that are plugged in through ADB.
+   *
+   * @param minApiLevel If <code>null</code>, all devices will be returned. Otherwise, only
+   *                      those device serials that are greater than or equal to the provided
+   *                      version will be returned.
+   */
+  public static Set<String> findAllDevices(AndroidDebugBridge adb, Integer minApiLevel) {
```

Comment:
1) please add a `@Nullable` on the minApiLevel param
2) please add an overloaded version of this method with the old signature.  It's part of the public API and I know some other folks are calling that version.